### PR TITLE
customize collabora a bit

### LINKED ISSUE #239: remove banner
Please remove this banner when creating a document from Nextcloud Office.
Benefit from the latest developments in online productivity. Nutzen Sie die Möglichkeiten kostenlos mit anderen im Browser. Apps are also available for Android and iOS. 21.11.1.3 brings major improvements in usability, visual presentation, and performance.

### PR DESCRIPTION
**Pull request by szaimen**
Close #239

Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

File: php/containers.json [hunk 1]
```diff
@@ -174,7 +174,8 @@
         "NC_DOMAIN=%NC_DOMAIN%",
         "TURN_SECRET=%TURN_SECRET%",
         "SIGNALING_SECRET=%SIGNALING_SECRET%",
-        "JANUS_API_KEY=%JANUS_API_KEY%"
+        "JANUS_API_KEY=%JANUS_API_KEY%",
+        "extra_params=--o:ssl.enable=false --o:welcome.enable=false"
```
[the remark this answers is not in the document]
Comment (szaimen) [reply php/containers.json:178]: Thanks! So do you recommmend setting ssl.termination to false?
https://github.com/CollaboraOnline/online/blob/2c458b1e289a049394ddccd9cd90c116e912bb72/coolwsd.xml.in#L145

[the remark this answers is not in the document]
Comment (szaimen) [reply php/containers.json:178]: Maybe like this:

extra_params=--o:ssl.enable=false --o:ssl.termination=true --o:user_interface.mode=classic --o:logging.level=warning

**Review by juliushaertl — APPROVED** [review]

**Comment by R7JANV1** [comment]
The banner is still displayed. If it cannot be removed, can the frequency of how often it is displayed be reduced?

**Comment by szaimen** [comment]
> can the frequency of how often it is displayed be reduced?

@juliushaertl is there a way to do that?

**Comment by juliushaertl** [comment]
Best would be to open an issue in https://github.com/CollaboraOnline/online for discussion about the frequency.

**Comment by szaimen** [comment]
@R7JANV1 feel free to do as advised!

**Pull request merged**